certificate-service/integration-test/src/test/java/se/inera/intyg/certificateservice/integrationtest/fk7211/FK7211ActiveIT.java

```diff
@@ -2664,6 +2665,28 @@ void shallReturn403IfUnitIsCareUnitAndNotOnCareUnit() {
       assertEquals(403, response.getStatusCode().value());
     }
 
+    @Test
+    @DisplayName("FK7211 - Om användaren har rollen barnmorska ska intyget gå att signeras")
+    void shallSuccessfullySignIfRoleIsMidwife() {
+      final var testCertificates = testabilityApi.addCertificates(
+          defaultTestablilityCertificateRequest(FK7211, VERSION)
+      );
+
+      final var response = api.signCertificate(
```

Comment:
Since the send-action also is affected by the role check, should we add a IT that verifies midwife can send certificate aswell?